### DISCUSSION TITLE
Generate an Adversarial Example

### DISCUSSION THREAD

**Discussion by AhmedHussKhalifa** [yoshitomo-matsubara/torchdistill#288] [Q&A]
Hey @yoshitomo-matsubara,

Can you walk me through how to generate an adversarial example within the dataloader, and what changes would need to be made to ensure that the adversarial samples are properly generated and integrated into the training process?

Thank you in advance.

**Comment by yoshitomo-matsubara** [comment]
Hi @AhmedHussKhalifa 

It sounds like nothing specific for torchdistill, and I would highly suggest you ask such questions at [PyTorch Discussion](https://discuss.pytorch.org/) instead.

**Reply by yoshitomo-matsubara** [reply]
@AhmedHussKhalifa 
You've already asked [a similar question here](https://github.com/yoshitomo-matsubara/torchdistill/discussions/171)

For follow-up questions, please use the same thread rather than starting a new thread, otherwise your question is treated as an answer by the system).

**Reply by yoshitomo-matsubara** [reply]
@AhmedHussKhalifa 
1. Please carefully read and close https://github.com/yoshitomo-matsubara/torchdistill/discussions/171 by marking the previous answer as answer
2. Based on my answers at https://github.com/yoshitomo-matsubara/torchdistill/discussions/171, re-post the question here  (again, as part of this thread, not as an "answer"
3. Delete duplicate "answers" [1](https://github.com/yoshitomo-matsubara/torchdistill/discussions/171#discussioncomment-5317020) and [2](https://github.com/yoshitomo-matsubara/torchdistill/discussions/288#discussioncomment-5306451)

**Comment by AhmedHussKhalifa** [comment]
Hey,
Sorry I was not clear.
Let's assume that I have an initialized object for an attack method in ./example/imageclassification.py, and I want to utilize it in BaseDatasetWrapper(). what changes would need to be made to BaseDatasetWrapper()?

**Reply by yoshitomo-matsubara** [reply]
@AhmedHussKhalifa 
I'll stop answering your questions as you don't use this space respectfully, and [this is repeated](https://github.com/yoshitomo-matsubara/torchdistill/discussions/282#discussioncomment-4977422). 

How can you
1. close this thread by marking your question as answer (I unmarked it), 
2. unmark the answer to [your previous question](https://github.com/yoshitomo-matsubara/torchdistill/discussions/171), and
3. [copy-paste your question as "answer" in the previous discussion page](https://github.com/yoshitomo-matsubara/torchdistill/discussions/171#discussioncomment-5317020) again, which is exactly what I asked you above not to do
?

**Comment by AhmedHussKhalifa** [comment]
Dear @yoshitomo-matsubara ,

I am writing to apologize for any misunderstandings that may have arisen in our previous interactions. As a new user of the discussion platform, I may have yet to understand the community's etiquette and norms of communication fully.

I understand that my previous questions may have come across as inconsiderate, and I am genuinely sorry for any misunderstanding of your instruction that I may have caused. I appreciate your patience in explaining your expectations to me, and I will do my best to follow them in the future.

Please know that I value your insights and contributions, and I hope to continue learning from you and others in this community. 

Please let me know if you accepted my apology. Thank you for taking the time to address my behaviour.
